static/js/publisher/release/actions/releases.test.js

```diff
@@ -1,4 +1,21 @@
-import { UPDATE_RELEASES, updateReleases } from "./releases";
+/* global global, jest */
```

Comment:
Isn't `jest` in globals by default in test env? I don't recall needing to add it.

Reply:
Might just be my editor, will remove and see if the linter complains!

Reply:
eslint failed without it